docs/DeveloperGuide.md

```diff
@@ -351,8 +351,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 ### Non-Functional Requirements
 
 1.  Should work on any _mainstream OS_ as long as it has Java `17` or above installed.
-2.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
-3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
+2.  Should work on Mac as long as javafx and java '17' both installed.
```

Comment:
Should this still be _mainstream OS_ since it is a requirement by 2103 to have the program run on all platforms?